Formula/wasmtime.rb

```diff
@@ -0,0 +1,28 @@
+class Wasmtime < Formula
+  desc "Standalone JIT-style runtime for WebAssembly, using Cranelift"
+  homepage "https://wasmtime.dev/"
+  license "Apache-2.0"
+  head "https://github.com/bytecodealliance/wasmtime.git"
+
+  stable do
+    url "https://github.com/bytecodealliance/wasmtime.git", tag: "v0.28.0"
+  end
```

Comment:
This doesn't need to be a separate block. If we are using a Git clone, we also need to specify `revision:` in addition to `tag:`. Please see the other formulae that do this.

Also, why do we need a git clone? Can we use a tarball?

Reply:
I suppose we can't use tarball because we need `git clone --recurse-submodules ..`. Building tarball won't work without it.

Reply:
What is in the submodules?

Reply:
Rust crates. The official build guide is:
```sh
git clone --recurse-submodules \
    https://github.com/bytecodealliance/wasmtime.git
cd wasmtime
cargo build --release
```

Reply:
Ok, if they're just Rust crates that's fine. If it's something else though we'll need to have a closer look at it, as we don't want this to vendor things that are provided by Homebrew.

Reply:
This builds only one binary `wasmtime`. Nothing more.

Reply:
It's not a question of what it builds. It's a matter of what the dependencies are, and there are some or else we wouldn't need submodules.

Reply:
From `git clone` output:
```text
Submodule 'crates/c-api/examples/wasm-c-api' (https://github.com/WebAssembly/wasm-c-api) registered for path 'crates/c-api/wasm-c-api'
Submodule 'WASI' (https://github.com/WebAssembly/WASI) registered for path 'crates/wasi-common/WASI'
Submodule 'crates/wasi-crypto/spec' (https://github.com/WebAssembly/wasi-crypto.git) registered for path 'crates/wasi-crypto/spec'
Submodule 'crates/wasi-nn/spec' (https://github.com/WebAssembly/wasi-nn) registered for path 'crates/wasi-nn/spec'
Submodule 'spec_testsuite' (https://github.com/WebAssembly/testsuite) registered for path 'tests/spec_testsuite'
```